seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/AbstractReadStrategy.java

```
@@ -144,6 +151,44 @@ public List<String> getFileNamesByPath(String path) throws IOException {
         return fileNames;
     }
 
+    private Date getFileModifiedDate(String modifiedDate) {
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+        if (modifiedDate != null) {
+            try {
+                return dateFormat.parse(modifiedDate);
+            } catch (ParseException e) {
+                log.warn(
+                        "Failed to parse file modified date format: yyyy-MM-dd HH:mm:ss, please check file_filter_modified_start or file_filter_modified_end format.");
+            }
```

we should throw exception instead of print an error.

---

@Hisoka-X  fixed.